lib/jekyll-feed/meta-tag.rb

```diff
@@ -17,21 +33,50 @@ def config
       @config ||= @context.registers[:site].config
     end
 
-    def attributes
+    def generator
+      @generator ||= @context.registers[:site].generators.select { |it| it.is_a? JekyllFeed::Generator }.first # rubocop:disable Metrics/LineLength
+    end
+
+    def link(collection, category)
+      attrs = attributes(collection, category).map { |k, v| %(#{k}="#{v}") }.join(" ")
```

Comment:
Since values from `attributes()` are always strings,
```suggestion
      attrs = attributes(collection, category).map { |k, v| "#{k}=#{v.inspect}" }.join(" ")
```

Reply:
Skipping this as I saw that this change was declined in #263.